release/pkg/assets_package_controller.go

```diff
@@ -16,20 +16,22 @@ package pkg
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 
 	anywherev1alpha1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
 )
 
 const (
-	packagesRootPath = "projects/aws/eks-anywhere-packages"
+	packagesRootPath  = "projects/aws/eks-anywhere-packages"
+	packagesImageName = "eks-anywhere-packages"
+	packagesHelmChart = "eks-anywhere-packages-helm"
```

Comment:
Is `eks-anywhere-packages-helm` the repository for the helm chart? I thought we were going to do that, but not sure if that was done yet

Reply:
Made some changes, good catch. LMK what you think about this format

Reply:
honestly would be nice if we had a repo like `eks-anywhere-packages/charts` and use the same versioning.

Reply:
> honestly would be nice if we had a repo like `eks-anywhere-packages/charts` and use the same versioning.

For the immediate term (bugbash) being this is probably not going to happen, but I'll bring it up during standup to discuss the pros/cons on this one.